src/utils/utils.js

```diff
@@ -525,36 +531,38 @@ const isDefinedAndNotNull = (x) => isDefined(x) && isNotNull(x);
 const getDataFromSource = (src, dest, properties) => {
   const data = {};
   if (isArray(src)) {
-    for (let index = 0; index < src.length; index += 1) {
-      if (properties[src[index]]) {
-        data[dest] = properties[src[index]];
+    // eslint-disable-next-line no-restricted-syntax
```

Comment:
Why are we using restricted syntax here? The previous "for" loop is fine.

Reply:
Revert these kinds of changes in other places too.

Reply:
reverted all relevant changes